src/target/llvm/codegen_llvm.cc

```diff
@@ -498,11 +498,12 @@ void CodeGenLLVM::AddAliasInfo(llvm::Instruction* inst, const VarNode* buffer, P
 void CodeGenLLVM::GetAlignment(DataType t, const VarNode* buf_var, const PrimExpr& index,
                                int* p_alignment, int* p_native_bits) {
   int max_align_bits = t.bits();
-  auto it = alloc_storage_info_.find(buf_var);
-  if (it != alloc_storage_info_.end()) {
-    const StorageInfo& info = it->second;
```

Comment:
Let us keep the original data structure info for now, in case we need to add more information later